src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/JiraSoftwareScenario.kt

```diff
@@ -45,6 +45,8 @@ class JiraSoftwareScenario : Scenario {
             jqlMemory = jqlMemory,
             issueKeyMemory = issueKeyMemory
         )
+        val findInitialIssues = ActionShuffler.findIssueKeysWithJql(jira, meter, issueKeyMemory)
```

Comment:
IMO it violates SRP. It doesn't seem to be the responsibility of shuffler to `findIssueKeysWithJql`.